android/java/AndroidManifest.xml

```diff
@@ -88,6 +88,14 @@
     android:screenOrientation="sensorPortrait"
     tools:ignore="LockedOrientationActivity,DiscouragedApi"/>
 
+<activity
+    android:name="org.chromium.chrome.browser.vpn.activities.VpnPaywallActivity"
```

Comment:
I don't understand why do you create a new `VpnPaywallActivity` here instead of modifying `BraveVpnPlansActivity`? It would better to rename the file if you prefer doing that as it would hold a history of modification of that file.

Reply:
@SergeyZhukovsky we need to keep the existing activity as we may need to use for different variant in future based on the pod discussion. i am going to remove BraveVpnPlansActivity if the tests look good.